src/bin/pgcopydb/ld_stream.h

```diff
@@ -101,6 +101,8 @@ typedef struct StreamContext
 	LogicalMessageMetadata metadata;
 	LogicalMessageMetadata previous;
 
+	uint64_t lastWrittenLSN;	/* LSN of last message written to the file */
+
 	uint64_t lastWrite;
```

Comment:
Add a comment that lastWrite is a timestamp, or change the name of the variable to `lastWriteTime`, or both.